### ISSUE TITLE
Mise à jour du header pour synchroniser avec la nouvelle version DSFR

### ISSUE THREAD

**Issue by lucaa** [DISIC/observatoire#1201]
![image](https://user-images.githubusercontent.com/657716/233661004-ea9f62f0-83db-4631-94c7-46b52aa96920.png)

à changer:
* nom du site à gauche: taille et gras de la police
* survol marqué en gris
* menu à droite:
  * titre du lien vers l'observatoire
  * gras du menu
  * plus de séparateur
  * padding au survol

**Comment by ClementNumericite** [comment]
Je ne sais plus si on avait statué sur les icônes une fois connecté @lucaa 
![Capture d’écran du 2023-04-24 18-43-13](https://user-images.githubusercontent.com/80679266/234062073-5db83c7b-9cdf-469a-ba49-bdd0283c9d31.png)

Il me semble qu'on avait évoqué la possibilité de les supprimer, mais nous n'en avons pas reparlé au dernier point.

**Comment by lucaa** [comment]
non, on n'en a pas parlé, mais t'as prévu de détecter la connexion sur la plateforme courante? Si oui, comment?

**Comment by ClemDee** [comment]
Les changements possibles ont été implémentés.

Par contre, le gras du menu à droite ne peut pas être implémenté, car c'est un gras moyen (500) correspondant à une police du nouveau dsfr que nous n'avons pas sur notre ancienne version. Notre niveau de gras supporté est un niveau 700, qui apparait donc beaucoup plus gras que ce qui est attendu. Pour l'instant j'ai implémenté le gras en `font-weight: 500`, qui du coup n'est pas implémenté et le texte s'affiche donc normalement.

**Comment by lucaa** [comment]
> Je ne sais plus si on avait statué sur les icônes une fois connecté @lucaa ![Capture d’écran du 2023-04-24 18-43-13](https://user-images.githubusercontent.com/80679266/234062073-5db83c7b-9cdf-469a-ba49-bdd0283c9d31.png)
> 
> Il me semble qu'on avait évoqué la possibilité de les supprimer, mais nous n'en avons pas reparlé au dernier point.

Après analyse, il semble que les menus de notifications et recherche sont affichés même pour les utilisateurs simples (non-admins).

J'ai crée les tickets #1206 et #1207 pour supprimer ces 2 menus.

Les notifications (#1206 ) sont probablement pas utilisés mais la recherche (#1207) est bien fonctionnelle et pourrait être utilisée par certains utilisateurs. L'accès à cette fonctionnalité restera disponible sur la page d'accueil.
